backend/api/image_generation.py

```
@@ -62,9 +64,21 @@ async def generate_scene_image(scene_description: str, scene_title: str, scenari
+            debug_log(f"[IMAGE] Temporary URL: {temp_image_url}")
             
-            # Return temporary URL directly (no local storage)
+            # Upload to Wasabi if scene_id is provided
+            if scene_id:
+                try:
+                    wasabi_url = await upload_scene_image_from_url(scene_id, temp_image_url)
+                    if wasabi_url and wasabi_url.strip():
+                        debug_log(f"[IMAGE] Uploaded to Wasabi: {wasabi_url}")
+                        return wasabi_url
+                    else:
+                        debug_log(f"[IMAGE] Wasabi upload failed, returning temporary URL")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)